main/src/cgeo/geocaching/utils/MapMarkerUtils.java

```diff
@@ -413,7 +413,7 @@ private static boolean showUserModifiedCoords(final Geocache cache) {
      *         True if the floppy overlay should be displayed
      */
     private static boolean showFloppyOverlay(@Nullable final CacheListType cacheListType) {
-        return cacheListType != CacheListType.OFFLINE; // also covers null check
```

Comment:
Just an unqualified question looking at the comment:
What about a NULL case?

Reply:
Ah, I simply erased the condition and always returned true for testing purposes. When could a `null` value happen - no list assigned? Is this a possible use case? (Search by address from mainscreen, maybe?)

Reply:
> Search by address from mainscreen, maybe?

Yes, maybe all the list generated from on demand like search results or lists of live map caches. But I am not deep enough into the code to judge....just guessing, why sometime in the past someone added this comment.

Reply:
added a null case check